lessons/Algebra2/exploring-linear-models/langs/en-us/pages/other-forms-linear-models.adoc

```diff
@@ -0,0 +1,77 @@
+= Other Forms of Linear Models
+++++
+<style>
+.studentAnswerMedium { min-width: 8em !important; }
+body.workbookpage .fitbruby { padding-top: 1.1rem; }
+body:not(.LessonPlan, .narrativepage) p { line-height: 1.2; }
+.center { padding: 0; }
+</style>
+++++
+
+For this page, you'll need to the @starter-file{alg2-states} open on your computer. If you haven't already, select **Save a Copy** from the "File" menu to make a copy of the file that's just for you.
+
+@n Evaluate @show{(code '(lr-plot states-table "state" "college-or-higher" "median-income"))}. What is @math{R^2}? @fitb{5em}{@ifsoln{}}
+
+@n Instead of using function notation, write the **Slope-Intercept form** of the optimal linear model below.
```

Comment:
I am confused by these directions. When students make the lr plot the information given reads as follows:
y=1142.03x+2.09e4; R: +0.76458; R-sqr: 0.58458

(Side note: looking carefully at the regression function, I notice that the y-intercept is 2.09e4. I would assume most students have no idea what that means. Wherever the first time is that they are seeing this notation, students/teachers would benefit from us offering some scaffolding for making sense of that.)

Reply:
Fixed.